src/iop/lowpass.c

```diff
@@ -533,7 +576,7 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pix
 #endif
     for(int k = 0; k < 0x10000; k++)
     {
-      float kx2m1 = 2.0f * (float)k / 0x10000 - 1.0f;
+      float kx2m1 = 2.0f * ((float)k / 0x10000 - d->cfulcrum / 100.0f);
```

Comment:
But then the algorithm does not keep history as here : 50/100 is not equal to 1.

Reply:
Also
float kx2m1 = 2.0f * (float)k / 0x10000 - 1.0f;
is not the same as
float kx2m1 = 2.0f * ((float)k / 0x10000 - 1.0f);

Reply:
> 
> 
> But then the algorithm does not keep history as here : 50/100 is not equal to 1.

You're right, but multiplied by 2 it does.

Reply:
> float kx2m1 = 2.0f * (float)k / 0x10000 - 1.0f;
> is not the same as
> float kx2m1 = 2.0f * ((float)k / 0x10000 - 1.0f);

You're right too, but
float kx2m1 = 2.0f * ((float)k / 0x10000 - 0.50f);
does.

Reply:
You're right, my bad!